content/applications/essentials/export_import_data.rst

```diff
@@ -476,3 +479,39 @@ The two files produced are ready to be imported in Odoo without any modification
 imported these two :abbr:`CSV (Comma-separated Values)` files, there are four contacts and three
 companies (the first two contacts are linked to the first company). Keep in mind to first import
 the companies, and then the people.
+
+Update data in Odoo
+===================
+
+Existing data can be updated in bulk through a data import, as long as the :ref:`External ID
+<essentials/external-id>` remains consistent.
+
+Prepare data export
+-------------------
+
+To update data through an import, first navigate to the data to be updated, and select the |list| to
+activate list view. On the far left side of the list, tick the checkbox for any record to be
+updated. Then, click |actions|, and select :icon:`fa-upload` :guilabel:`Export` from the drop-down
+menu.
+
+On the resulting :guilabel:`Export Data` pop-up window, tick the checkbox labeled :guilabel:`I want
+to update data (import-compatible export)`. This automatically includes the External ID in the
+export, and limits the :guilabel:`Fields to export` list to **only** include fields that are able to
+be imported. Select the required fields to be included in the export using the :ref:`options
+<export-data>` on the pop-up window, then click :guilabel:`Export`.
+
+.. note::
+   The :guilabel:`External ID (id)` field does **not** appear in the :guilabel:`Fields to export`
+   list unless it is manually added. However, if the :guilabel:`I want to update data
+   (import-compatible export)` checkbox is ticked, it is included in the export.
```

Comment:
```suggestion
.. note::
   The :guilabel:`External ID` field does **not** appear in the :guilabel:`Fields to export`
   list unless it is manually added, but it is still included in the export. However, if the :guilabel:`I want to update data
   (import-compatible export)` checkbox is ticked, it is included in the export.
```
This suggestion to your admonition would clarify things I brought up in the previous comment. Does this still ring true to the statement you are trying to convey? Or does this suggestion veer off-track from what you are trying to explain? Just want to make sure we're delivering the most comprehensible documentation possible, so as to avoid any potential confusion with readers/users. Just something to consider during your revisions. 👍 

Also, when I manually add 'External ID' to that list, there is no "(id)" parenthetical that follows. So, that might be worth a quick double-check, as well.

![Screenshot 2024-07-23 083320](https://github.com/user-attachments/assets/56d3ba3a-222e-4a19-af66-8757bc9645ca)

Reply:
I believe the (id) is not included because you did not have dev mode on, so it was not showing the technical name for the field. I will update